CSV/specifications.csv

```diff
@@ -91,7 +91,7 @@ QT,QT,"The <a href='http://www.apple.com/quicktime/' target='_blank'>QuickTime</
 RealHD,RealHD,<a href='http://www.realnetworks.com/realmediahd'>RealMedia™ HD</a>
 ross,Ross,<a href='http://www.rossvideo.com/'>Ross Video</a>
 samsung,Samsung,"<a href='http://www.samsung.com/'>Samsung</a> "
-SCTE-215-1-1b, SCTE 215-1-1 2020b,"<a href='https://www.scte.org/download-scteisbe-standards/'>SCTE Specifications</a>"
+SCTE-215-1-1b,SCTE 215-1-1 2020b,"<a href='https://www.scte.org/download-scteisbe-standards/'>SCTE</a> HEVC Video Constraints for Cable Television Part 1-1 HDR10 Coding"
```

Comment:
Let's have the same shortname in columns one and two here (and match the suggestion above)?

Reply:
I thought column 2 could be the name of the spec and have spaces (e.g. like other entries such as ISO-CENC, ISO Common Encryption"

Reply:
It was intended to be used like that but it looks like the first column is never used and the second one is used to reference the spec. I can fix it and make a new PR for that.